src/topology/metric_space/basic.lean

```diff
@@ -202,7 +202,7 @@ by simp [nndist, edist_dist, real.to_nnreal, max_eq_left dist_nonneg, ennreal.of
 lemma edist_nndist (x y : α) : edist x y = ↑(nndist x y) :=
 by { rw [edist_dist, nndist, ennreal.of_real_eq_coe_nnreal] }
 
-@[simp, norm_cast] lemma ennreal_coe_nndist (x y : α) : ↑(nndist x y) = edist x y :=
+@[simp, norm_cast] lemma coe_nnreal_ennreal_nndist (x y : α) : ↑(nndist x y) = edist x y :=
```

Comment:
```suggestion
@[simp, norm_cast] lemma coe_ennreal_nndist (x y : α) : ↑(nndist x y) = edist x y :=
```
the `nnreal` seems superflous here, since `nndist` always returns `nnreal`.

Reply:
I agree it's not strictly necessary here, but having a completely uniform naming convention helps a lot (the previous state of affairs was a random mess where each statement could be justified with some argument, but this made many statement names unpredictable). So I'd rather keep the name as it is. I'll change if you insist, of course.

Reply:
Fine by me, just wanted to check this was intentional and not a side effect of a bulk change applied mechanically.